prompt-service/src/unstract/prompt_service/authentication_middleware.py

```diff
@@ -25,6 +25,8 @@ def validate_bearer_token(token: Optional[str]) -> bool:
             cursor = db.execute_sql(query)
             result_row = cursor.fetchone()
             cursor.close()
+            # Closes the used connection without returning it to the pool.
```

Comment:
What's the purpose of doing this @harini-venkataraman if we wish to use the connection pool?

Reply:
@chandrasekharan-zipstack Addressed it [here](https://github.com/Zipstack/unstract/pull/670#discussion_r1753881150).